source/Calamari/Integration/Scripting/WindowsPowerShell/Bootstrap.ps1

```diff
@@ -246,13 +247,13 @@ Log-VersionTable
 # -----------------------------------------------------------------
 # Variables
 # -----------------------------------------------------------------
-
+{{BeforeVariablesDebugLocation}}
 {{VariableDeclarations}}
```

Comment:
Think these are necessary? Feels like it would be to debug our stuff - I think the use case for this feature is just to debug their stuff.